web/src/components/BarcodeChart.vue

```diff
@@ -161,13 +164,17 @@ export default class BarcodeChart extends Vue {
 </script>
 
 <style lang="scss">
-@use 'codeHighlightsColours';
+@use 'scssVariables';
```

Comment:
```suggestion
@use 'variables';
```
The fact we're using scss is implied.